modules/accredited_representative_portal/app/sidekiq/accredited_representative_portal/power_of_attorney_form_submission_job.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyFormSubmissionJob
+    class PendingSubmissionError < StandardError; end
+
+    include Sidekiq::Job
+    include SentryLogging
+
+    sidekiq_options retry_for: 48.hours
+
+    attr_reader :response
+
+    def perform(poa_form_submission_id)
+      @id = poa_form_submission_id
+      service = BenefitsClaims::Service.new(poa_form_submission.power_of_attorney_request.claimant.icn)
+      @response = service.get_2122_submission(poa_form_submission.service_id)
+      poa_form_submission.update(
+        status: new_status,
+        service_response: response.to_json,
+        status_updated_at: DateTime.current,
+        error_message: error_data.to_json
+      )
+      raise PendingSubmissionError, '2122 still pending' if response_status == 'pending'
+    rescue => e
+      handle_errors(e, poa_form_submission)
+    end
+
+    sidekiq_retries_exhausted do |job, _ex|
+      poa_form_submission_id = job['args'].first
+      poa_form_submission = PowerOfAttorneyFormSubmission.find(poa_form_submission_id)
+      poa_form_submission.update(status: :failed, status_updated_at: DateTime.current)
```

Comment:
We need to document next steps somewhere for exhausted jobs: either an alternate path for resubmission or escalating the issue through an alert. Once expiration is implemented, the next steps will need to be taken within 60 days.